scripts/report-bundle-size/report-as-a-pr-comment.js

```diff
@@ -134,18 +138,20 @@ async function updateOrAddComment(difference, resultsBaseQuery, localBundleSizes
 }
 
 function createMessage(difference, resultsBaseQuery, localBundleSizes) {
-  let message = '| 📦 Bundle Name| Base Size | Local Size | 𝚫% | Status |\n| --- | --- | --- | --- | :---: |\n'
+  let message =
+    '| 📦 Bundle Name| Base Size | Local Size | 𝚫 | 𝚫% | Status |\n| --- | --- | --- | --- | --- | :---: |\n'
   let highIncreaseDetected = false
   difference.forEach((diff, index) => {
     const baseSize = formatSize(resultsBaseQuery[index].size)
     const localSize = formatSize(localBundleSizes[diff.name])
+    const diffBytes = formatSize(diff.change)
```

Comment:
**🥜 nitpick:** At first I thought that was an unformated bytes count. Suggestion: ‏rename it to `diffSize` to match the same pattern as `baseSize` and `localSize`, so we know it's formatted similarly